FreetimeTests/IssueCommentTableTests.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  IssueCommentTableTests.swift
+//  FreetimeTests
+//
+//  Created by B_Litwin on 6/6/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+import XCTest
+import StyledText
+@testable import Freetime
+
+class IssueCommentTableTests: XCTestCase {
+    
+    func test_tableColumnDimensions() {
+        let contentSizeCategory = UIApplication.shared.preferredContentSizeCategory
```

Comment:
Just for readability, can we pull out all this setup logic into the `setUp` or similar? Makes the test a bit easier to read and scan.